### PR TITLE
deleted PRS8423Salomon

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/1831

### CODE REVIEW

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: PRS8001-9000/PRS8422Salomon.xml [hunk 1]
```diff
@@ -48,46 +48,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sameAs="wd:Q343719" sex="1">
-
-
+            <person sameAs="wd:Q717219" sex="1">
                   <persName xml:lang="gez" xml:id="n1" type="birth">ሰሎሞን፡</persName>
                   <persName xml:lang="gez" corresp="#n1" type="normalized">Salomon II</persName>
```
Comment (eu-genia) [inline PRS8001-9000/PRS8422Salomon.xml:53]: I wonder if "II" should be marked up as addName
even `<addName>II</addName> (or <addName>III</addName>)`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS8001-9000/PRS8422Salomon.xml [hunk 2]
```diff
@@ -48,46 +48,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sameAs="wd:Q343719" sex="1">
-
-
+            <person sameAs="wd:Q717219" sex="1">
                   <persName xml:lang="gez" xml:id="n1" type="birth">ሰሎሞን፡</persName>
```
Comment (eu-genia) [inline PRS8001-9000/PRS8422Salomon.xml:52]: Salomon is the regnal name

File: PRS8001-9000/PRS8422Salomon.xml [hunk 3]
```diff
@@ -48,46 +48,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                   <persName xml:lang="gez" corresp="#n1" type="normalized">Salomon II</persName>
                   <persName xml:lang="gez" xml:id="n2" type="birth">ገብረ፡ መድሕን፡</persName>
```
Comment (eu-genia) [inline PRS8001-9000/PRS8422Salomon.xml:54]: here type="birth" is correct

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: PRS8001-9000/PRS8422Salomon.xml [hunk 4]
```diff
@@ -48,46 +48,41 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sameAs="wd:Q343719" sex="1">
-
-
+            <person sameAs="wd:Q717219" sex="1">
```
[the remark this answers is not in the document]
Comment (thea-m) [reply PRS8001-9000/PRS8422Salomon.xml:51]: now I'm completely confused :D
I was working under the assumption PRS8422Salomon=Salomon II, 1796-1799=Q717219 (Salomon III in wikipedia)
and PRS8421Salomon=Salomon I, 1777-1779=Q343719 (Salomon II in wikipedia)

**Review by eu-genia — APPROVED** [review]

**Pull request merged**